kafka/adminapi.go

```diff
@@ -1555,6 +1642,11 @@ func (a *AdminClient) SetSaslCredentials(username, password string) error {
 
 // Close an AdminClient instance.
 func (a *AdminClient) Close() {
+	var newValue uint32 = 1
+	currentValue := atomic.LoadUint32((*uint32)(unsafe.Pointer(&a.isClosed)))
```

Comment:
While this code is logical, I'm not sure about the cast to 4 byte int. Size of boolean in go (at least on my machine) is only 1 byte, and not 4. Since there is no CAS for booleans, maybe we can use uint8 as the type of isClosed, with a comment. 
Other alternatives also welcome on this, can't figure out what is the 100% correct way.